docs/src/tutorials/conic/dualization.jl

```diff
@@ -0,0 +1,218 @@
+# Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors    #src
+# This Source Code Form is subject to the terms of the Mozilla Public License   #src
+# v.2.0. If a copy of the MPL was not distributed with this file, You can       #src
+# obtain one at https://mozilla.org/MPL/2.0/.                                   #src
+
+# # Dualization
+
+# The purpose of this tutorial is to explain how to use [Dualization.jl](@ref) to
+# improve the performance of some conic optimization models. There are two
+# important takeaways:
+#
+#  1. JuMP reformulates problems to meet the input requirements of the
+#     solver, potentially increasing the problem size by adding slack variables
+#     and constraints.
+#  2. Solving the dual of a conic model can be more efficient than solving the
+#     primal.
+
+# This tutorial uses the following packages
+
+using JuMP
+import Dualization
+import SCS
+
+# ## Background
+
+# Conic optimization solvers typically accept one of two input formulations.
+
+# The first is the _standard_ conic form:
+# ```math
+# \begin{align}
+#     \min_{x \in \mathbb{R}^n} \; & c^\top x \\
+#               \;\;\text{s.t.} \; & A x = b  \\
+#                               & x \in \mathcal{K}
+# \end{align}
+# ```
+# in which we have a set of linear equality constraints $Ax = b$ and the
+# variables belong to a cone $\mathcal{K}$.
+#
+# The second is the _geometric_ conic form:
+# ```math
+# \begin{align}
+#     \min_{x \in \mathbb{R}^n} \; & c^\top x \\
+#               \;\;\text{s.t.} \; & A x - b \in \mathcal{K}
+# \end{align}
+# ```
+# in which an affine function $Ax - b$ belongs to a cone $\mathcal{K}$ and the
+# variables are free.
+
+# It is trivial to convert between these two representations, for example, to go
+# from the geometric conic form to the standard conic form we introduce slack
+# variables $y$:
+# ```math
+# \begin{align}
+#     \min_{x \in \mathbb{R}^n} \; & c^\top x \\
+#               \;\;\text{s.t.} \; & [A -I] [x; y] = b \\
+#                               & [x; y] \in \mathbb{R}^n \times \mathcal{K}
+# \end{align}
+# ```
+# and to go from the standard conic form to the geometric conic form, we can
+# rewrite the equality constraint as a function belonging to the `{0}` cone:
+# ```math
+# \begin{align}
+#     \min_{x \in \mathbb{R}^n} \; & c^\top x \\
+#               \;\;\text{s.t.} & [A; I] x - [b; 0] \in \{0\} \times \mathcal{K}
+# \end{align}
+# ```
+
+# From a theoretical perspective, the two formulations are equivalent, and if
+# you implement a model in the standard conic form and pass it to a geometric
+# conic form solver (or vice versa), then JuMP will automatically reformulate
+# the problem into the correct formulation.
+
+# From a practical perspective though, the geometric to conic reformulation is
+# particularly problematic, because the additional slack variables and
+# constraints can make the problem much larger and therefore harder to solve.
```

Comment:
The standard to geometric is as problematic. It's confusing to say one is more problematic because everything that is happening on the primal side also appears in the dual side.
For instance, if you have `x in K` that you convert to geometric, you need to add `dimension(K)` free variables and a constraint in `K` while if you dualize, you don't need to add any free variable, only a constraint in `K`.